comfort/other/SimpleSynchronizer.lua

```diff
@@ -0,0 +1,448 @@
+---
```

Comment:
ich habe tatsächlich schon nen MPSyncer in der communitylib. merge?

Reply:
Der sieht unnötig kompliziert aus.